blog-service/2023-04-28-apps.md

```diff
@@ -0,0 +1,20 @@
+---
+title: April 21, 2023 (Apps)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - apps
+  - app catalog
+  - saas cloud
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Asana App for Sumo Logic. This app leverages Sumo Logic's [Cloud-to-Cloud Asana source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source) to provide real-time monitoring and security insight for your Asana account. Here are some of the key features that this app offers:
+
+* **Customizable searches**. Build custom searches to examine particular Asana data elements, allowing you to go deeper into the data and find insights that are pertinent to your company's needs.
```

Comment:
Id' rephrase this as:

**Integration with other sources**.  Combine with other Sumo Logic apps to gain a more holistic view of the data.

Reply:
updated.